runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorkerTest.java

```diff
@@ -3067,8 +3069,14 @@ public void testExceptionInvalidatesCache() throws Exception {
 
       assertThat(
           // The commit will include a timer to clean up state - this timer is irrelevant
-          // for the current test. Also remove source_bytes_processed because it's dynamic.
-          setValuesTimestamps(commit.toBuilder().clearOutputTimers().clearSourceBytesProcessed())
+          // for the current test. Also remove source_bytes_processed and
+          // per_work_item_latecy_attributions because they're dynamic.
```

Comment:
ie you could extract to a helper method
removeDynamicFields(CommitWorkRequest request) 

and use that here and other places instead of dropping in the "server"

Reply:
Done.